src/app/admin/_utils/mapIdArrayToIdObjects.js

```diff
@@ -0,0 +1,5 @@
+function mapIdArrayToIdObjects(IDs) {
```

Comment:
🟠 please use camel case for args
🟠 what is the point of this function anyway? :) name is kinda confusing a bit, to be sure you need to go and check out code, which kills whole point of abstracting this action...and if you don't want to put that map each time, there is something similar in `lodash` package i think 🤔

Reply:
The point of this function is to map array of IDs)

I put it away into the file where it is being used.